homeassistant/components/tplink/strings.json

```diff
@@ -25,6 +25,42 @@
       "no_devices_found": "[%key:common::config_flow::abort::no_devices_found%]"
     }
   },
+  "entity": {
+    "sensor": {
+      "current_consumption": {
+        "name": "Current consumption"
+      },
+      "total_consumption": {
+        "name": "Total consumption"
+      },
+      "today_consumption": {
+        "name": "Today's consumption"
+      }
+    },
+    "switch": {
+      "led": {
+        "name": "LED"
+      },
+      "plug_1": {
+        "name": "Plug 1"
+      },
+      "plug_2": {
+        "name": "Plug 2"
+      },
+      "plug_3": {
+        "name": "Plug 3"
+      },
+      "plug_4": {
+        "name": "Plug 4"
+      },
+      "plug_5": {
+        "name": "Plug 5"
+      },
+      "plug_6": {
+        "name": "Plug 6"
+      }
+    }
```

Comment:
I'll link an issue where there was a decision made on translations looking like "xxx 1". But even tho renaming devices work, automatic translations would still be a lot better imo

Reply:
I agree to disagree given the fact that people provision these devices using the app while also giving them rememberable names. Requiring them to perform renaming once again (and again, in case they need to start again with a clean installation) only for homeassistant would greatly hinder the initial UX, especially for people who may have dozens of devices.

Also, wouldn't this also be a breaking change, _if_ (I'm not sure if that's the case though) these translated names would override the existing friendly names currently read from the device? Entity ids should stay the same, but it's a small consolation if all your devices would suddenly need to be renamed manually.

Reply:
So wait, let me try to understand. It is possible to set the device alias via their app?

Reply:
Yes, of course, see https://www.youtube.com/watch?v=THHIjPAhpuM (starting from 2min or so) for an idea how it looks like in the app.

Reply:
Aaaah, that makes sense. I thought the names were always Plug1, Plug2, Plug3 like in the testdata, so I was confused since I really dislike that kind of naming